docker-compose.yml

```diff
@@ -50,3 +50,22 @@ services:
       - db
     links:
       - db:db
+  luigi:
+    build:
+      context: .
+      dockerfile: Dockerfile.luigi
+    env_file: .env
+    # 2G Limit
+    mem_limit: 2147483648
+    environment:
+      MYSQL_HOST: db
+    volumes:
+      - ./data-imports/:/root/data-imports/
+      - ./data/:/root/heatseek/
+      - ./luigi/:/root/luigi/
+    ports:
+      - "8082:8082"  # default luigid port
```

Comment:
Whoops. Left over from when I was actually starting luigid up and needed to expose a port.

This PR is not really the right way to do things, it's just the first thing I got working. I figured it was better to get something out there / get feedback than get something totally polished.

I didn't go with luigid here b/c I'm still a little confused about the central scheduler and wanted to get something checked in. The model is described here: http://luigi.readthedocs.io/en/stable/execution_model.html
I don't understand if the best approach is to create one container that runs luigid, then create other containers for actually running the jobs (as workers)? Or to just docker-compose exec commands right on luigid?

Thinking about it more, I'm pretty sure it's the former, but then you've got multiple python containers running. However, if I understand the linked doc correctly, the luigid server would only need to install luigi as requirement, and doesn't need the packages required to run specific jobs. Additionally, instead of the "LUIGI_WORKERS" env setting here, you could just use docker-compose scale to spin up multiple worker containers all running the same job. I'll try this out and come back with the results.